server/paths/api.ts

```diff
@@ -18,4 +20,7 @@ export default {
     },
     show: coursePath,
   },
+  referrals: {
+    index: referralsPath,
```

Comment:
I suppose. It would be the same path if we had an index though. Would we duplicate the path with an `index` key if we had one? The path/route distinction in Express is a little different to Rails 🤔

Reply:
Yeah I think we probably would duplicate them to distinguish from posting to create and getting multiple results back